nixos/modules/services/networking/freeradius.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.freeradius;
+
+    freeradiusService = cfg: optionalAttrs cfg.enable {
+    freeradius = {
+      description = "FreeRadius server";
+      wantedBy = ["multi-user.target"];
+      after = ["network-online.target"];
+      requires = ["network-online.target"];
+      preStart = ''
+        ${cfg.configDir}/certs/bootstrap
```

Comment:
Is the user supposed to provide that script?

Reply:
This is some kind of typo that I forgot to remove.